api/customObjects.ts

```diff
@@ -12,7 +12,7 @@ const SCHEMA_API_PATH = 'crm-object-schemas/v3/schemas';
 export function batchCreateObjects(
   accountId: number,
   objectTypeId: string,
-  objects: JSON
+  objects: Schema
```

Comment:
This endpoint looks like it takes a list of object definitions. The [api catalogue](https://developers.hubspot.com/docs/reference/api/crm/objects/objects#post-%2Fcrm%2Fv3%2Fobjects%2F%7Bobjecttype%7D%2Fbatch%2Fcreate) shows an example of what it looks like. I also believe the shape for the objects will look a little different than the shape of the schemas.